templates/container-kill-template.yml

```diff
@@ -0,0 +1,16 @@
+---
+variables:
+  APP_NS: default
+  APP_LABEL: "run=nginx"
+  ##only deployment is supported
+  APP_KIND: deployment 
+  TARGET_CONTAINER: "nginx"
+  EXPERIMENT_IMAGE: litmuschaos/ansible-runner:latest
+  OPERATOR_IMAGE: litmuschaos/chaos-operator:latest
+  RUNNER_IMAGE: litmuschaos/chaos-runner:latest
+
+.container_kill_template:
+  image: uditgaurav/gitlab-template:latest
```

Comment:
we need a mayadata-io image for this. Dockerfile too can be placed (I suppose in ci-lib repo)